extractor/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInName/config-example1-expected.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name = "Checker">
+    <property name="charset" value="UTF-8"/>
+
+    <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+    <property name="severity" value="warning"/>
+
+    <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+    <property name="haltOnException" value="false"/>
+
+    <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="module\-info\.java$" />
+    </module>
+
+    <module name="TreeWalker">
+        <module name="AbbreviationAsWordInName"></module>
```

Comment:
I think expected is multiple lines module with close tag, please make it as it s generated

Reply:
Done. If i understood the comment right, i kept it as it is. However, i had to spend a lot of time trying to deal with whitespaces being created in the generated xml, which was making the test cases fail despite the expected and generated looking similar.

Reply:
Did you resolve problem or need help?

Reply:
So, far it seems to be resolved. It did take some time though. Parsing XML can be quite tricky given my experience so far.